app/views/layouts/application.html.erb

```diff
@@ -3,7 +3,7 @@
   <head>
     <title><%= Setting.site_name %></title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
```

Comment:
It seems like this and the other font-related change are closely related. Could you put them on the same branch and open a new pull request?